tee/src/lib.rs

```diff
@@ -285,49 +259,51 @@ pub mod pallet {
 		/// If the enclave is assigned, it will be placed in queue for tech committee approval
 		/// If enclave is not already assigned, he can exit without permission.
 		#[pallet::weight(T::WeightInfo::unregister_enclave())]
-		pub fn unregister_enclave(
-			origin: OriginFor<T>,
-		) -> DispatchResultWithPostInfo {
+		pub fn unregister_enclave(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
 			let account = ensure_signed_or_root(origin)?;
 			match account {
 				Some(account_id) => {
-					let enclave_id  = EnclaveIndex::<T>::get(account_id.clone()).unwrap();
+					let enclave_id = EnclaveIndex::<T>::get(account_id.clone()).unwrap();
 
-					let un_reg_enclaves:  BoundedVec<EnclaveId, T::MaxUnRegisteredEnclaves> = <UnRegisteredEnclaves<T>>::get();
+					let un_reg_enclaves: BoundedVec<EnclaveId, T::MaxUnRegisteredEnclaves> =
+						<UnRegisteredEnclaves<T>>::get();
 
-					let reg_enclaves  = <RegisteredEnclaves<T>>::get();
+					let reg_enclaves = <RegisteredEnclaves<T>>::get();
 
-					if!reg_enclaves.contains(&enclave_id){
+					if !reg_enclaves.contains(&enclave_id) {
 						EnclaveRegistry::<T>::remove(enclave_id);
-						Self::deposit_event(Event::UnregisterEnclave { account_id,   enclave_id });
+						Self::deposit_event(Event::UnregisterEnclave { account_id, enclave_id });
 					} else {
-						let un_reg_enclaves = un_reg_enclaves
-							.try_mutate(|v| v.push(enclave_id))
-							.unwrap();
+						let un_reg_enclaves =
+							un_reg_enclaves.try_mutate(|v| v.push(enclave_id)).unwrap();
 						<UnRegisteredEnclaves<T>>::put(un_reg_enclaves);
-						Self::deposit_event(Event::MovedForUnregistration { account_id,   enclave_id });
+						Self::deposit_event(Event::MovedForUnregistration {
+							account_id,
+							enclave_id,
+						});
 					}
-				}
-				_ => {}
+				},
+				_ => {},
 			}
 
-
 			Ok(().into())
 		}
 
 		// TODO:  Fix this
```

Comment:
Will this TODO be fixed later?

Reply:
This is ok, it's on the Tee branch and will be update but not by @Immanuel-john

Reply:
Thank you